Description: Don't disable vmport even if we're using spice, as this
 prevents having a seamless mouse when using Ubuntu as a guest.
Origin: reverts https://github.com/virt-manager/virt-manager/commit/ddfca74e54a7638844b72baf3d30502dcf59b251
Forwarded: No

Index: virt-manager-1.4.1/virtinst/guest.py
===================================================================
--- virt-manager-1.4.1.orig/virtinst/guest.py	2017-03-09 18:11:22.184898076 +0100
+++ virt-manager-1.4.1/virtinst/guest.py	2017-03-09 18:11:22.180898114 +0100
@@ -874,12 +874,6 @@
             else:
                 self.features.pae = self.capsinfo.guest.supports_pae()
 
-        if (self.features.vmport == "default" and
-            self.os.is_x86() and
-            self.has_spice() and
-            self.conn.check_support(self.conn.SUPPORT_CONN_VMPORT)):
-            self.features.vmport = False
-
         if (self._os_object.is_windows() and
             self._hv_supported() and
             self.conn.check_support(self.conn.SUPPORT_CONN_HYPERV_VAPIC)):
